pkg/flare/archive.go

```diff
@@ -82,6 +83,7 @@ func CompleteFlare(fb flaretypes.FlareBuilder, diagnoseDeps diagnose.SuitesDeps)
 
 	if pkgconfigsetup.SystemProbe().GetBool("system_probe_config.enabled") {
 		fb.AddFileFromFunc(filepath.Join("expvar", "system-probe"), getSystemProbeStats) //nolint:errcheck
+		fb.AddFileFromFunc("system_probe_telemetry.txt", getSystemProbeTelemetry)        // nolint:errcheck
```

Comment:
should we check for an error here?

Reply:
I think it would make sense to gather up all the errors from these functions and try to write them to one last file at the end, but currently the flare logic collectively suppresses errors

Reply:
Turns out they get logged to flare_creation.log